src/extension.ts

```diff
@@ -102,6 +102,11 @@ export function activate(context: vscode.ExtensionContext) {
                 vscode.window.showInformationMessage('Stack Protocol Server is starting, please try again later!');
             }
         });
+        
+        vscode.commands.registerCommand('adapter.restartserver',  async () => {
+            const serverId: any = await vscode.window.showQuickPick(serversData.servers.map((server: any) => ({label: server.id})), {placeHolder: 'Select runtime/server to restartserver'});
```

Comment:
`serversData.servers` is not array anymore, it is an instance of Map<string, Protocol.SrverHandle>. It does not have map function. See other actions for example.